Source/WebCore/page/Frame.h

```diff
@@ -91,7 +91,6 @@ class Frame : public ThreadSafeRefCounted<Frame, WTF::DestructionThread::Main>,
     inline RefPtr<HTMLFrameOwnerElement> protectedOwnerElement() const; // Defined in HTMLFrameOwnerElement.h.
 
     WEBCORE_EXPORT void disconnectOwnerElement();
-    NavigationScheduler& navigationScheduler() const { return m_navigationScheduler.get(); }
```

Comment:
I don't think we should remove this one. It can remain useful for cases where we need to use the NavigationScheduler more than once in the scope of a function:
```
CheckedPtr navigationScheduler = frame->navigationScheduler();
if (navigationScheduler->shouldDoFoo())
    navigationScheduler->doFoo();
```

Reply:
That's a good point, will undo that change before landing. Thanks!